packages/front-end/components/Features/RuleModal.tsx

```diff
@@ -68,6 +73,24 @@ export default function RuleModal({
       header={rules[i] ? "Edit Override Rule" : "New Override Rule"}
       submit={form.handleSubmit(async (values) => {
         const ruleAction = i === rules.length ? "add" : "edit";
+
+        if (values.scheduleRules.length) {
```

Comment:
This will need to be updated if we want to support more than a start date and end date.